acsys/dpm/__init__.py

```diff
@@ -69,7 +69,7 @@ def isReadingFor(self, *tags):
         warnings.warn(
             "deprecated in favor of the snake_case version, is_reading_for",
             DeprecationWarning)
-        return self.is_reading_for
+        return self.is_reading_for()
```

Comment:
Should we pass `*tags`? What does the empty parens do?

Reply:
Possibly, I don't think it matters since no user should ever call this.

Reply:
To say more, because this is in a "private" class `_ItemCommon`, this only defines the shape of the inherited classes, so the argument signature is what needs to be correct.
Notice that the function that is called doesn't use the `tags` argument.